### PR TITLE
[COMP-LABELS] Cleanup and fix composable-labels model changeset

### PR DESCRIPTION
**Pull request by lampajr**
## Changes proposed

- [x] Remove unnecessary `public` schema
- [x] Remove `ALTER OWNER` instructions
- [x] Add `GRANT ALL` to the created sequences
- [x] Gave a bit more meaningful names to foreign keys 

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by lampajr** [comment]
This is still not enough.. there are still some missing permission somewhere

File: horreum-backend/src/main/resources/db/changeLog.xml [hunk 1]
```diff
@@ -4609,175 +4609,119 @@
...
+            ALTER TABLE ONLY exp_label
+                ADD CONSTRAINT fkv5kkr3oh8fnwd0uj0k684jda FOREIGN KEY (targetgroup_id) REFERENCES exp_labelgroup(id);
+
+            ALTER TABLE ONLY exp_label_values
+                ADD CONSTRAINT fkchg0rpfmrxc0y1vbcm21gcqe0 FOREIGN KEY (run_id) REFERENCES exp_run(id);
+            ALTER TABLE ONLY exp_label_values
+                ADD CONSTRAINT fkteh5nire3ba4hd6nn23vjrjok FOREIGN KEY (label_id) REFERENCES exp_label(id);
+
+            ALTER TABLE ONLY exp_run
+                ADD CONSTRAINT fkfre0ttv8g1062vrmvxtjd208y FOREIGN KEY (test_id) REFERENCES exp_labelgroup(id);
```
[the remark this answers is not in the document]
Comment (willr3) [reply horreum-backend/src/main/resources/db/changeLog.xml:4724]: I'm fine with changeSet entries changing until they are merged to master / main, then they should probably be stable.

[the remark this answers is not in the document]
Comment (lampajr) [reply horreum-backend/src/main/resources/db/changeLog.xml:4724]: yeah I agree, once merged it should be changed at all!

**Comment by lampajr** [comment]
> This is still not enough.. there are still some missing permission somewhere

Ok now the permissions should be properly set.

Tests are working and the changset execution works with a prod backup as well.

**Pull request merged**